src/img/icons/three-dots-row.svg

```diff
@@ -1,11 +1,6 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 21.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="Warstwa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 width="16px" height="16px" viewBox="0 0 16 16" enable-background="new 0 0 16 16" xml:space="preserve">
-<path d="M9,4.78C9,4.9,8.903,4.997,8.783,4.997H7.217C7.097,4.997,7,4.9,7,4.78V3.215c0-0.12,0.097-0.217,0.217-0.217h1.565
-	C8.903,2.997,9,3.095,9,3.215V4.78z"/>
-<path d="M9,8.78C9,8.9,8.903,8.997,8.783,8.997H7.217C7.097,8.997,7,8.9,7,8.78V7.215c0-0.12,0.097-0.217,0.217-0.217h1.565
-	C8.903,6.997,9,7.095,9,7.215V8.78z"/>
-<path d="M9,12.785c0,0.12-0.097,0.217-0.217,0.217H7.217C7.097,13.003,7,12.906,7,12.785V11.22c0-0.12,0.097-0.217,0.217-0.217
-	h1.565C8.903,11.003,9,11.1,9,11.22V12.785z"/>
-</svg>
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 30 30" style="enable-background:new 0 0 30 30;" xml:space="preserve">
+<circle cx="15" cy="3" r="3"/>
+<circle cx="15" cy="15" r="3"/>
+<circle cx="15" cy="27" r="3"/>
+</svg>

```

Comment:
No way. Please convert these to `path`s before merging. Not all browsers know how to render native elements, so the safest route is to convert

Reply:
Thanks @rdig I had not seen this issue before. After looking into it briefly, it is a worth avoiding. Thank you pointing it out. I will update.

Reply:
Updated to use paths now. It reduced file size too.